common/src/main/java/com/wynntils/utils/type/BoundingShape.java

```diff
@@ -1,44 +1,40 @@
 /*
- * Copyright © Wynntils 2023.
+ * Copyright © Wynntils 2023-2024.
  * This file is released under LGPLv3. See LICENSE for full license details.
  */
 package com.wynntils.utils.type;
 
 public interface BoundingShape {
```

Comment:
This would btw be a perfect candidate for the new `sealed` keyword:

```
public sealed interface BoundingShape permits BoundingBox, BoundingCircle, BoundingPolygon {
```

 It's saying that the only possible BoundingShapes are Boxes, Circles and Polygons. This can help the compiler generate more efficient code in some cases, and it clarifies the logic. Not saying you must, just pointing out that this is one of the few examples I've ever seen where it would make sense.